papers/mass_spec_encoder.md

```diff
@@ -0,0 +1,82 @@
+---
+author: Adrian Mirza
+date: 2024-06-09
+title-block-banner: 
+bibliography: ../references.bib
+---
+
+
+# An End-to-End Deep Learning Framework for Translating Mass Spectra to De-Novo Molecules
+
+
+## Why discuss this paper? 
+
+I chose to discuss this particular paper because of two fundamental aspects:
+
+- Predicting experimental spectra using Machine Learning is historically a difficult and interesting task.
+- Our groups are interested in this research topic.
```

Comment:
```suggestion
I chose to discuss this particular paper because of two fundamental aspects:

- Predicting experimental spectra using Machine Learning has historically been a difficult and interesting task.
- Our groups are interested in this research topic.
```